packages/core/src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -387,7 +387,7 @@ class AuthFlowAuthorization extends SsoAccessTokenProvider {
     override async registerClient(): Promise<ClientRegistration> {
         const companyName = getIdeProperties().company
         return this.oidc.registerClient({
-            clientName: isCloud9() ? `${companyName} Cloud9` : `${companyName} Toolkit for VSCode`,
+            clientName: isCloud9() ? `${companyName} Cloud9` : `${companyName} IDE Extensions for VSCode`,
```

Comment:
Let's add a comment for future devs, too.

```suggestion
            // All AWS extensions (Q, Toolkit) for a given IDE use the same client name.
            clientName: isCloud9() ? `${companyName} Cloud9` : `${companyName} IDE Extensions for VSCode`,
```